ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+#    Please do not ask general questions via Issues    #
+
+Instead, find community resources at https://graphql.org/community/
+
+#    Please do not make feature requests via Issues    #
```

Comment:
Seems like we really don't want people using the "Issues" section.

I can get behind that, but PRs increase the barrier of entrance to the strawman stage.

Reply:
Yeah, Issues get ignored much more than PRs, the quality of Issues are typically below that of PRs, and Issues are often drive by requests from people who do not actually intend on following through while those opening PRs tend to follow up :)

I'll think of how to word this a little better - but some amount of creating a barrier for strawman is intentional since we already have ~50 strawman without clear champions in the issues section